provider/terraform.rb

```diff
@@ -40,6 +40,12 @@ def tf_type(property)
       tf_types[property.class]
     end
 
+    # "Namespace" - prefix with product and resource - a property with
+    # information from the "object" variable
+    def namespace_property_from_object(property, object)
+       "#{object.__product.prefix[1..-1].camelize}#{object.name}#{property.name.camelize}"
```

Comment:
nit: Would it be a huge pain to have these start with a lowercase letter? I know our var/fn visibility is all over the map and nothing is going to get used outside the google package anyway, but it still feels wrong to have these uppercase.

Reply:
Nope. Done!